tests/php/Forms/GridField/GridFieldTest.php

```diff
@@ -659,4 +661,68 @@ public function testValidationMessageInOutput()
         $gridfieldOutput = $gridField->FieldHolder();
         $this->assertStringNotContainsString('<p class="message ' . ValidationResult::TYPE_ERROR . '">', $gridfieldOutput);
     }
+
+    public function testAddRequestToURL()
+    {
+        $gridField = new GridField('testfield', 'testfield');
+        $link = Controller::join_links('class-name', 'item', '1');
+
+        $request = new HTTPRequest(
+            'POST',
+            'admin/gridfield/1',
```

Comment:
Don't use hardcoded IDs in tests. If these represent the IDs of a fixture, get the object from the fixture and then get the ID from the object.

Otherwise, if these aren't IDs of fixtures, please explain what is happening here.

Reply:
This URL doesn't have any special purpose. I can replace it with any value without ID. Is it OK?

Reply:
Yeah, having something that's clearly just an abitrary value is better. Even if you just put a comment there that says something like "the actual URL path here is arbitrary"

Reply:
DONE